File: app/src/main/java/com/riders/thelab/data/local/model/CitiesEventJson.java
package com.riders.thelab.data.local.model;

import com.riders.thelab.data.remote.dto.weather.City;

import java.util.List;

import lombok.Getter;
import lombok.ToString;

/**
 * Moshi implementation (Gson library replacement)
 * <p>
 * Moshi is a modern JSON library for Android and Java. It makes it easy to parse JSON into Java objects:
 * <p>
 * Reference : https://github.com/square/moshi
 */
@Getter
@ToString
public class CitiesEventJson {

    List<City> citiesList;
}

File: app/src/main/java/com/riders/thelab/data/local/model/weather/CitiesEventJson.kt
package com.riders.thelab.data.local.model.weather


import com.riders.thelab.data.remote.dto.weather.City
import lombok.Getter

/**
 * Moshi implementation (Gson library replacement)
 * <p>
 * Moshi is a modern JSON library for Android and Java. It makes it easy to parse JSON into Java objects:
 * <p>
 * Reference : https://github.com/square/moshi
 */
@Getter
data class CitiesEventJson(val citiesList: List<City>) {
}